### PR TITLE
feat: migrate pages from converia

### PR DESCRIPTION
**Pull request by maehr**
# Pull request

- Migrate pages from <https://conferences.unibas.ch/frontend/index.php?folder_id=232&page_id=>

## Types of changes

- [ ] New feature (non-breaking change which adds functionality).
- [x] Enhancement (non-breaking change which enhances functionality)
- [ ] Bug Fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [x] I have read the **[README](./README.md)** document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have mentioned all co-authors in the PR description as `Co-authored-by: Name <email@domain.com>`.

## Release Notes

- **New Features**
  - Comprehensive updates to the README for improved installation and usage instructions.
  - Introduction of new documents for the conference, including "Terms and Conditions," "Call for Contributions," and detailed program schedules.
  - Enhanced navigation structure for a streamlined user experience.
  - New structured document for the "Book of Abstracts" detailing conference presentations.

- **Documentation**
  - Significant revisions to the about.qmd to include essential information about the organizing committee, sponsors, and code of conduct.
  - New sections in various documents detailing submission guidelines, evaluation criteria, and logistical information for attendees.

- **Style**
  - Updates to the website title and navigation items for clarity and conciseness.
  - Introduction of a new CSS class for button styling.

### CODE REVIEW

**Comment by mtwente** [comment]
I had a quick look around:

- [ ] the sponsoring logo file on `about.html` is still fetched from Converia (but host logo file is not). Some hyperlinks point to converia, too.
- [ ] lists are sometimes not formatted as such (e.g. OC members on `home`, but on `about` it works)
- [ ] internal hyperlinks do not work (missing `.html` in the end of the URL), e.g. all links on `conference-program`
- [ ] schedule only shows time+chair: we could add the presentations for each session, linking to the abstracts
- [ ] make sure all image alt-texts have been migrated properly
- [ ] sub-items of sidebar menu item `Conference Program` are ordered counter-intuitively (schedule should come first)
- [ ] is the separate [keynote page](https://conferences.unibas.ch/frontend/index.php?folder_id=315&page_id=) missing on purpose?
- [ ] imprint/privacy policy from converia have not been migrated. Do you want an exact copy of converia or does it make more sense to create a new imprint?
- [ ] on a similar note: keep present tense/switch to past tense? Keep preliminary/wrong room allocation or update to de-facto rooms?
- [ ] explicit contact info is missing (basically the right column on converia)
- [ ] all abstracts will link to zenodo. We could additionally include a sentence about the Zenodo community and/or the slides that were (in some cases) published along side. 
  - maybe slide linking on the abstracts could be improved, ATM it's almost invisible using quarto features

**Comment by maehr** [comment]
> I had a quick look around:
> 
> * [x]  the sponsoring logo file on `about.html` is still fetched from Converia (but host logo file is not). Some hyperlinks point to converia, too.
> * [x]  lists are sometimes not formatted as such (e.g. OC members on `home`, but on `about` it works)
> * [x]  internal hyperlinks do not work (missing `.html` in the end of the URL), e.g. all links on `conference-program`
> * [x]  schedule only shows time+chair: we could add the presentations for each session, linking to the abstracts
> * [ ]  make sure all image alt-texts have been migrated properly
> * [x]  sub-items of sidebar menu item `Conference Program` are ordered counter-intuitively (schedule should come first)
> * [x]  is the separate [keynote page](https://conferences.unibas.ch/frontend/index.php?folder_id=315&page_id=) missing on purpose?
> * [x]  imprint/privacy policy from converia have not been migrated. Do you want an exact copy of converia or does it make more sense to create a new imprint?
> * [x]  on a similar note: keep present tense/switch to past tense? Keep preliminary/wrong room allocation or update to de-facto rooms?
> * [x]  explicit contact info is missing (basically the right column on converia)
> * [ ]  all abstracts will link to zenodo. We could additionally include a sentence about the Zenodo community and/or the slides that were (in some cases) published along side.
>   
>   * maybe slide linking on the abstracts could be improved, ATM it's almost invisible using quarto features

Actionable Follow-ups:

- [x] Link Sessions to Schedule: Update the schedule to link each session to the appropriate detailed session page.
- [ ] Image ALT Texts: Review all images to confirm that alt-texts are accurately migrated and relevant.
- [x] Keynote Page Addition: Add the keynote page as required.
- [x] Imprint Decision: Clarify whether to create a new imprint or replicate the Converia version as is.
- [x] Consistent Language Use: Keep all content in the present tense.

File: conference-program/schedule.qmd [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Schedule
+---
+
+## Thursday, 12 September 2024
+
+| Time     | Lecture Hall 001                                | Lecture Hall 115                                              |
+|----------|-------------------------------------------------|---------------------------------------------------------------|
+| 09:00 am | **Conference Opening**                          |                                                               |
+|          | Chair(s): Prof. Dr. Martin                      |                                                               |
```
Comment (mtwente) [inline conference-program/schedule.qmd:10]: Some of the names did not make it into the file:
* Opening: Lucas, Christiane, Enrico are missing;
* Closing: Béatrice is missing

 @maehr Do you want full names incl. academic titles + affiliation?

Comment (maehr) [reply conference-program/schedule.qmd:10]: Thanks. Yes, we should definitely add opening and closing. No opinion on the titles.

**Review by mtwente — APPROVED** [review]

File: styles.css [hunk 2]
```diff
@@ -96,6 +96,22 @@
...
+	background-color: #a5d7d2 !important;
+	border-color: #a5d7d2 !important;
+	border-radius: 0.25rem !important;
+	&:hover {
+		color: #181818 !important;
+		text-decoration: underline;
+		text-decoration-color: #ff3cc7;
+		text-decoration-thickness: 3px;
+	}
+}
```
[the remark this answers is not in the document]
Comment (maehr) [reply styles.css:113]: @mtwente does this make sense?

[the remark this answers is not in the document]
Comment (mtwente) [reply styles.css]: I was just being lazy. I have updated the css class, now it works without `!important`.

**Pull request merged**